packages/extension-sdk-react/README.md

```diff
@@ -0,0 +1,137 @@
+# Extension SDK For React
+
+## Installation
+
+Add dependency to your project using yarn or npm
+
+`yarn add @looker/extension-sdk-react`
+
+or
+
+`npm install @looker/extension-sdk-react`
+
+## Usage
+
+The Extension SDK for react contains of a provider that allows child components to access the
+Extension SDK.
+
+### Provider
+Add the `ExtensionProvider` near the root of the extension component tree.
+```
```

Comment:
tag languages for syntax highlighting? https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks